src/store/dashboard/awsDashboard/awsDashboardWidgets.ts

```diff
@@ -38,6 +38,7 @@ export const computeWidget: AwsDashboardWidget = {
     computedReportItem: ComputedReportItemType.usage,
     computedReportItemValue: ComputedReportItemValueType.total,
     titleKey: messages.dashboardDailyUsageComparison,
+    chartName: 'dashboardDailyUsageComparison',
```

Comment:
Would you mind prefixing the chart names with "aws"; for example, "awsComputeChart"